rosidl_typesupport_c/include/rosidl_typesupport_c/message_type_support_dispatch.h

```diff
@@ -24,6 +24,17 @@ extern "C"
 {
 #endif
 
+/// Get the message type support handle specific to this identifier
+/**
+ * This function is a general function used to be called by a generated message's c functionality
+ * If the identifier is the same as this handle's typesupport_identifier, then the handle is
+ * simply returned, otherwise it's loaded from a shared library.
+ *
+ * If no type support handle can be found, this function returns NULL.
```

Comment:
The return value should be documented with `\return` - not in text.

Reply:
Fixed here and below.